test-suite/output/Notations4.out

```diff
@@ -29,6 +29,11 @@ match e with
 | _ => [e + e]
 end
      : Expr -> Expr
+File "./output/Notations4.v", line 61, characters 0-56:
+Warning: Notation "_" in custom expr was already used.
```

Comment:
this is an error not a warning https://gitlab.com/coq/coq/-/jobs/1909045561#L3235